javascript/statements.json

```diff
@@ -2128,6 +2128,45 @@
           }
         }
       },
+      "using_keyword": {
```

Comment:
Why not just `using`?

```suggestion
      "using": {
```

Reply:
Yes, sorry, this was generated from an outdated local collector version.